api/proto/external.proto

```diff
@@ -255,7 +255,10 @@ message RingMLSAG {
 message SignatureRctBulletproofs {
     repeated RingMLSAG ring_signatures = 1;
     repeated CompressedRistretto pseudo_output_commitments = 2;
-    bytes range_proofs = 3;
+    bytes range_proof_bytes = 3;
```

Comment:
Could we add documentation here about the new fields to help external consumers of this API? (that can be a follow-on PR if needed)

Reply:
👍

Reply:
e.g. from rct_bulletproofs for `range_proof_bytes` `This field is empty if mixed transactions are enabled`

Reply:
I think Matt omitted all the doc strings on these initially, because no one is really using these structs directly, they only handle `Tx` usually, and these are just constituents inside that.

But maybe there are people who are making their own transaction builders or something who will use these structs directly.